src/main/java/seedu/address/logic/commands/person/ImportCommand.java

```diff
@@ -53,8 +53,12 @@ public class ImportCommand extends Command {
             + PREFIX_IMPORT + "./data/patient.csv";
```

Comment:
Could change this to "./RapidTracerData/PatientData.csv"

Reply:
Actually, the data directory(where the addressbook.json file is) is auto generated when the app is launched but the RapidTracerData directory is only generated if export command is used.